### PR TITLE
Jordan/pro 204 change founders

### PR DESCRIPTION
**Pull request by blokboy**

### CODE REVIEW

**Comment by tbtstl** [comment]
Can you include the deploy script as well?

**Comment by tbtstl** [comment]
Also, this looks like it overwrote the metadata renderer implementation – it should have overwrote the token implementation. Did the wrong contract get deployed or was the script writing to the wrong file?

File: script/DeployTokenUpgrade.s.sol [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+        vm.startBroadcast(deployerAddress);
+
+        // Deploy root manager implementation + proxy
+        Manager manager = Manager(managerProxy);
+
+        // Deploy token upgrade implementation
+        address tokenUpgradeImpl = address(new Token(managerProxy));
+
+        address managerImpl = address(new Manager(tokenImpl, tokenUpgradeImpl, auctionImpl, treasuryImpl, governorImpl));
```
Comment (tbtstl) [inline script/DeployTokenUpgrade.s.sol:57]: you're passing in tokenImpl twice here

File: deploys/5.upgradeMetadata.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
-Metadata Renderer implementation: 0x1d613cdfbd3bbe7bc7715b90b002cce1454b8bef
-Manager implementation: 0xff44c385de21b32d5de0cc7a918499ea5631ea52
+Metadata Renderer implementation: 0x5d48d87eccc97d5a59be048e128d9ab2458edcc7
```
Comment (tbtstl) [inline deploys/5.upgradeMetadata.txt:1]: this file should be reverted

File: script/DeployTokenUpgrade.s.sol (hunk 1 above)
Comment (blokboy) [reply script/DeployTokenUpgrade.s.sol:57]: oh good catch

**Pull request merged**